docs/tutorials/controlling-wedo2-motor.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Controlling a WeDo 2.0 motor
+subject: Networking
+author: "@JorgePe"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+LEGO Education released the second version of WeDO in the beginning of 2016.
+We'll show how to use the bluez, the linux bluetooth stack, to wireless control a
+WeDo 2.0 motor.
+
+The first WeDO version uses USB so every robot needs to be tethered to a «host»
+(usually a computer but can also be a Mindstorms EV3 running ev3dev)
+The second WeDO version uses BLE (Bluetooth Low Energy, a sub-set of the Bluetooth
+4.0 standard) so robots can now be totally autonomous.
+
+## Requirements
+
+The EV3 internal bluetooth is to old, just 2.1 compliant. To use BLE we need an
+USB Bluetooth 4.0 dongle supported by ev3dev. If it works with Ubuntu or with a
+Raspberry Pi then most probably will also work with ev3dev.
+
+Since we'll probably use Wi-Fi, an USB hub will be required aswell. Most (but not
+all) USB 2.0 hubs work fine with ev3dev.
+
+If ev3dev recognizes our Bluetooth 4.0 dongle, we'll have two hci devices - the
+internal bluetooth and the new USB one:
+
+    robot@ev3dev:~# hciconfig -a
+    hci1:   Type: BR/EDR  Bus: UART
+	    BD Address: 00:17:EC:48:44:6B  ACL MTU: 1021:4  SCO MTU: 180:4
+       	    UP RUNNING 
+            RX bytes:863 acl:0 sco:0 events:32 errors:0
+    	    TX bytes:1396 acl:0 sco:0 commands:32 errors:0
+	    Features: 0xff 0xff 0x2d 0xfe 0x9b 0xff 0x79 0x83
+	    Packet type: DM1 DM3 DM5 DH1 DH3 DH5 HV1 HV2 HV3 
+	    Link policy: RSWITCH HOLD SNIFF PARK 
+	    Link mode: SLAVE ACCEPT 
+	    Name: 'ev3dev'
+	    Class: 0x000000
+	    Service Classes: Unspecified
+	    Device Class: Miscellaneous, 
+	    HCI Version: 2.1 (0x4)  Revision: 0x0
```

Comment:
As I said above, on newer EV3s, this will say `HCI Version: 4.0 (0x6)  Revision: 0x0` here, so we need to clarify this a bit.

Reference: <http://botbench.com/blog/2016/01/10/ev3-bricks-ble-or-blah/>